ova/generate_ova.sh

```diff
@@ -267,42 +184,31 @@ main() {
     if [ -z "${CHECKSUM_DIR}" ]; then
         CHECKSUM_DIR="${OUTPUT_DIR}"
     fi
-    # Check if versions of main packages have been specified
-    if  [ "${HAVE_VERSION}" = true ] && [ "${HAVE_ELK_VERSION}" = true ] && [ "${HAVE_OPENDISTRO_VERSION}" = true ]; then
-       
-        if [ ${BRANCH} != ${BRANCHDOC} ] && [ ${PACKAGES_REPOSITORY} = "prod" ]; then
-            echo "Wazuh branch must be equal to Documentation branch in production."
-            clean 1
-        fi
-
-        export UI_REVISION="${UI_REVISION}"
-        check_version
-
-        OVA_VERSION="${WAZUH_VERSION}_${OPENDISTRO_VERSION}"
-        [[ ${PACKAGES_REPOSITORY} = "prod" ]] && REPO="production" || REPO="development"
-        echo "Version to build: ${WAZUH_VERSION}-${OPENDISTRO_VERSION} with ${REPO} repository and ${BRANCH} branch"
-        
-        # Build OVA file (no standard)
-        build_ova ${WAZUH_VERSION} ${OVA_VERSION} ${BRANCH} ${BRANCHDOC} ${DEBUG}
 
-        # Standarize OVA
-        bash setOVADefault.sh "${scriptpath}" "${OUTPUT_DIR}/${OVA_VM}" "${OUTPUT_DIR}/${OVA_VM}" "${scriptpath}/wazuh_ovf_template" "${WAZUH_VERSION}" "${OPENDISTRO_VERSION}" || clean 1
-        
-        if [ "${CHECKSUM}" = "yes" ]; then
-            mkdir -p ${CHECKSUM_DIR}
-            cd ${OUTPUT_DIR} && sha512sum "${OVA_VM}" > "${CHECKSUM_DIR}/${OVA_VM}.sha512"
-            echo "Checksum created in ${CHECKSUM_DIR}/${OVA_VM}.sha512"
-        fi
-
-        echo "Process finished"
-        clean 0
-
-    else
-        echo "ERROR: Need more parameters."
-        help 1
+    [[ ${PACKAGES_REPOSITORY} = "prod" ]] && REPO="production" || REPO="development"
+
+    UNATTENDED_URL="https://raw.githubusercontent.com/wazuh/wazuh-packages/${PACKAGES_BRANCH}/resources/open-distro/unattended-installation/unattended-installation.sh"
+    WAZUH_VERSION=$(curl -s ${UNATTENDED_URL} | grep "WAZUH_VER=" | cut -d "\"" -f 2)
+    OPENDISTRO_VERSION=$(curl -s ${UNATTENDED_URL} | grep "OD_VER=" | cut -d "\"" -f 2)
```

Comment:
Doesn't makes sense to me, use the files that are already at this repo + branch.

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/832/commits/9d4132b545b62ad4399d421d3e37db7dc3a73ea2